liquibase/changeLogs/capture/data/aqToNwisParm/aqToNwisParm.tsv

```diff
@@ -0,0 +1,731 @@
+parameter	parm_cd
```

Comment:
Note to ourselves, maybe we want to build a loader for these parm codes, and also one for the stat codes.